pages/linux/gcov.md

```diff
@@ -0,0 +1,29 @@
+# gcov
+
+> Code coverage analysis and profiling tool. Discover untested parts of your program.
+> Get copy of source code annotated with execution frequencies of code segments.
+> More information: <https://gcc.gnu.org/onlinedocs/gcc/Invoking-Gcov.html>.
+
+- Generate coverage report named `file.cpp.gcov`:
+
+`gcov {{path/to/file.cpp}}`
+
+- Write individual execution counts for every basic block:
+
+`gcov --all-blocks {{path/to/file.cpp}}`
+
+- Write branch frequencies to output file and print summary information to `stdout` as a percentage:
+
+`gcov --branch-probabilities {{path/to/file.cpp}}`
+
+- Write branch frequencies as the number of branches taken, rather than the percentage:
+
+`gcov --branch-counts {{path/to/file.cpp}}`
+
+- Do not create `gcov` output file:
```

Comment:
Perhaps this will read better?
```suggestion
- Do not create a `gcov` output file:
```

Also, what kind of file is this? Is it just 1 of many output files, or the only one?